entry/src/entry.rs

```diff
@@ -111,15 +111,24 @@ pub struct Api<'a> {
 /// field points to Transactions that took place shortly before `hash` was generated.
 ///
 /// If you multiply `num_hashes` by the amount of time it takes to generate a new hash, you
-/// get a duration estimate since the last Entry. Since processing power increases
+/// get a duration estimate since the last `Entry`. Since processing power increases
 /// over time, one should expect the duration `num_hashes` represents to decrease proportionally.
 /// An upper bound on Duration can be estimated by assuming each hash was generated by the
 /// world's fastest processor at the time the entry was recorded. Or said another way, it
 /// is physically not possible for a shorter duration to have occurred if one assumes the
 /// hash was computed by the world's fastest processor at that time. The hash chain is both
 /// a Verifiable Delay Function (VDF) and a Proof of Work (not to be confused with Proof of
 /// Work consensus!)
-
+///
+/// The solana core protocol currently requires an `Entry` to contain `transactions` that are
+/// executable in parallel (implemented in
+/// `solana_core::banking_stage::BankingStage::process_and_record_transactions()`). All transactions
```

Comment:
I know these method references are ugly, but since the actual implementations are so hard to track and follow, I thought and additional breadcrumb would be nice. Again, I'm open to more general references to BankingStage and ReplayStage.

Reply:
I've added the TVU link and turned it into a list, to make it more readable:

![image](https://user-images.githubusercontent.com/694419/219531666-e2a6db66-9a00-4768-ac9a-12d20ff9127a.png)